RELEASE_NOTES.rst

```diff
@@ -21,6 +21,271 @@
 
 .. towncrier release notes start
 
+Airflow 2.6.0 (2023-04-20)
+--------------------------
+
+Significant Changes
+^^^^^^^^^^^^^^^^^^^
+
+Default permissions of file task handler log directories and files has been changed to "owner + group" writeable (#29506).
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  Default setting handles case where impersonation is needed and both users (airflow and the impersonated user)
+  have the same group set as main group. Previously the default was also other-writeable and the user might choose
+  to use the other-writeable setting if they wish by configuring ``file_task_handler_new_folder_permissions``
+  and ``file_task_handler_new_file_permissions`` in ``logging`` section.
+
+SLA callbacks no longer add files to the dag processor manager's queue (#30076)
+"""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  This stops SLA callbacks from keeping the dag processor manager permanently busy. It means reduced CPU,
+  and fixes issues where SLAs stop the system from seeing changes to existing dag files. Additional metrics added to help track queue state.
+
+The ``cleanup()`` method in BaseTrigger is now defined as asynchronous (following async/await) pattern (#30152).
+""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""
+  This is potentially a breaking change for any custom trigger implementations that override the ``cleanup()``
+  method and uses synchronous code, however using synchronous operations in cleanup was technically wrong,
+  because the method was executed in the main loop of the Triggerer and it was introducing unnecessary delays
+  impacting other triggers. The change is unlikely to affect any existing trigger implementations.
```

Comment:
```suggestion
This is potentially a breaking change for any custom trigger implementations that override the ``cleanup()``
method and uses synchronous code, however using synchronous operations in cleanup was technically wrong,
because the method was executed in the main loop of the Triggerer and it was introducing unnecessary delays
impacting other triggers. The change is unlikely to affect any existing trigger implementations.
```